### DISCUSSION TITLE
Freelancer Websites

### DISCUSSION THREAD

**Discussion by marcialwushu** [DevComputaria/Forum#4] [General]
Uma lista de websites para conseguir freelancing

1. Fiverr
2. Upwork
3. Toptal
4. Simply Hired
5. PeoplePerHour
6. Aquent
7. Crowded
8. The Creative Group
9. 99Designs
10. Nexxt
11. Writer Access
12. TaskRabbit
13. Skyword
14. Designhill
15. Freelancer
16. Guru
17. Hireable
18. FlexJobs

**Comment by marcialwushu** [comment]
Eu adicionaria ainda nessa lista: workana, getninjas, 99freelas, vintepila

**Reply by AndreNuernberg** [reply]
@marcialwushu você já escutou sobre a Vibbra!? É uma plataforma muito boa, um pouco diferente da proposta das demais. A Vibbra consegue ter uma equipe que fica mais próxima dos profissionais e ajuda no processo da relação entre profissional e cliente.

https://www.vibbra.com.br/vibbrantes/
